### PR TITLE
Try Sparse Checkout with https

### PR DESCRIPTION
**Pull request by scbedd**

### CODE REVIEW

File: eng/common/pipelines/templates/steps/sparse-checkout.yml [hunk 1]
```diff
@@ -38,8 +38,8 @@ steps:
             } else {
               Write-Host "Repository $($repository.Name) is being initialized."
 
-              Write-Host "git clone --no-checkout --filter=tree:0 git://github.com/$($repository.Name) ."
-              git clone --no-checkout --filter=tree:0 git://github.com/$($repository.Name) .
+              Write-Host "git clone --no-checkout --filter=tree:0 https://github.com/$($repository.Name) ."
+              git clone --no-checkout --filter=tree:0 https:git://github.com/$($repository.Name) .
```
Comment (weshaggard) [inline eng/common/pipelines/templates/steps/sparse-checkout.yml:42]:
```suggestion
              git clone --no-checkout --filter=tree:0 https://github.com/$($repository.Name) .
```

**Review by weshaggard — APPROVED** [review]

File: eng/common/pipelines/templates/steps/sparse-checkout.yml (hunk 1 above)
Comment (scbedd) [reply eng/common/pipelines/templates/steps/sparse-checkout.yml]: I saw this! corrected the syntax haha.

**Comment by scbedd** [comment]
This works. Going to be merging across the board with the [sync tools pr ](https://github.com/Azure/azure-sdk-for-python/pull/21535).

**Pull request closed without merging**